spec/test_templates/yaml/iam_passrole_resource_wildcard/iam_inline_policy_passrole_resource_wildcard_pass.yml

```diff
@@ -29,5 +29,31 @@ Resources:
             Effect: Allow
             Action: "iam:PassRole"
             Resource: "arn:aws:s3:::*"
+      Groups:
+        - !Ref GenericGroup
+
+  InlinePolicyDenyPass:
+    Type: "AWS::IAM::Policy"
+    Properties:
+      PolicyName: WildcardDenyResourcePolicy
+      PolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          -
+            Effect: "Allow"
+            Action:
+              - "s3:ListBucket"
+              - "s3:GetBucketLocation"
+            Resource: "arn:aws:s3:::*"
+          -
+            Effect: Allow
+            Action:
+              - "s3:ListBucket"
+              - "s3:GetBucketLocation"
+            Resource: "*"
+          -
+            Effect: Deny
```

Comment:
In general rules that check for things like * resource, etc. should likely ignore the Deny. i.e. this will still trigger the `lib/cfn-nag/custom_rules/IamPolicyWildcardResourceRule.rb` rule.  Otherwise we are properly following the changes requested for PassRole rules themselves.